samples/encoder/main.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * This is an example designed to work with the charge controller,
```

Comment:
An example shouldn't be targeted at specific hardware. Here, obviously, you are using specific hardware for this test, but the explanation should just talk about what you're trying to test and what success is supposed to look like